README.md

```diff
@@ -224,6 +224,77 @@ permanent: true
 The permanent and runtime settings are independent, so you can set only the runtime, or only the permanent.  You cannot
 set both permanent and runtime to `false`.
 
+Examples of Options
+-------------------
+By default, any changes will be applied immediately, and to the permanent settings. If you want the changes to apply immediately but not permanently, use `permanent: no`. Conversely, use `runtime: no`.
+
+Permit TCP traffic for port 80 in default zone:
+
+```yaml
+firewall:
+  - port: 80/tcp
+    state: enabled
+```
+
+Do not permit TCP traffic for port 80 in default zone:
+
+```yaml
+firewall:
+  - port: 80/tcp
+    state: disabled
+```
+
+Add masquerading to dmz zone:
+
+```yaml
+firewall:  
+  - masquerade: yes
+    zone: dmz
+```
+
+Remove masquerading to dmz zone:
+
+```yaml
+firewall:
+  - masquerade: no
+    zone: dmz
```

Comment:
`state` is a required parameter - please add `state: disabled` here

Reply:
I think the state would have to be set to `state: enabled here` since masquerading cannot be used with the `state` being set to `disabled`. https://github.com/linux-system-roles/firewall/blob/94e0c6dde38200f3db33ae069388a3075ee1a7d9/library/firewall_lib.py#L402